lxd/daemon.go

```diff
@@ -798,7 +798,7 @@ func (d *Daemon) Init() error {
 	/* Read the uid/gid allocation */
 	d.IdmapSet, err = shared.DefaultIdmapSet()
 	if err != nil {
-		shared.LogWarn("Error reading default idmap", log.Ctx{"err": err.Error()})
+		shared.LogWarn("Error reading default uid/gid map", log.Ctx{"err": err.Error()})
```

Comment:
Unless there are a lot of things to show and the order is unimportant of course.